2015-04-20 15:16:00,024  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred during query, discarding reggie [com.sun.jini.reggie.RegistrarProxy[registrar=bb59ab03-2e81-4720-8995-a6d218ac57e5 DynamicSmartStub [ImplObjClass: com.sun.jini.reggie.GigaRegistrar, ConnectionURL: NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar], MaxConnPool: 1024 ]]]; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.ConnectException: Connection refused
2015-04-20 15:16:04,772  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred during query, discarding reggie [com.sun.jini.reggie.RegistrarProxy[registrar=bb59ab03-2e81-4720-8995-a6d218ac57e5 DynamicSmartStub [ImplObjClass: com.sun.jini.reggie.GigaRegistrar, ConnectionURL: NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar], MaxConnPool: 1024 ]]]; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.ConnectException: Connection refused
2015-04-20 15:16:05,270  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred while attempting to register with the lookup service event mechanism; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.ConnectException: Connection refused
2015-04-20 15:16:10,196  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred while attempting to register with the lookup service event mechanism; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.ConnectException: Connection refused
2015-04-20 15:16:15,198  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred while attempting to register with the lookup service event mechanism; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.ConnectException: Connection refused
2015-04-20 15:16:20,195  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred while attempting to register with the lookup service event mechanism; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.ConnectException: Connection refused
2015-04-20 15:16:25,199  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred while attempting to register with the lookup service event mechanism; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.ConnectException: Connection refused
2015-04-20 15:16:30,200  INFO [net.jini.lookup.ServiceDiscoveryManager] - Exception occurred while attempting to register with the lookup service event mechanism; Caused by: java.rmi.ConnectException: Connect Failed to [NIO://kafka.local:54556/pid[40994]/1429517374140307232_2_-706347898903840095_details[class com.sun.jini.reggie.GigaRegistrar]]; nested exception is: 
	java.net.ConnectException: Connection refused
2015-04-20 15:16:39,591  INFO [com.gigaspaces.localcache] - LocalCache#1-myGrid was created successfully. Total time: 1.2 seconds
